### PR TITLE
Add oscar dummy file to flow generator

### PR DESCRIPTION
**Pull request by NGoetz**
This adds dummy file in the oscar format for the flow generator.

### CODE REVIEW

File: src/particle-analysis/GenerateFlow.py [hunk 1]
```diff
@@ -217,9 +217,53 @@ def generate_dummy_JETSCAPE_file(self,output_path,number_events,multiplicity,see
                                    self.py_[particle],self.pz_[particle]))
 
             output.write("#	sigmaGen	0.0	sigmaErr	0.0")
+            
+    def generate_dummy_OSCAR_file(self,output_path,number_events,multiplicity,seed):
+            """
+            Generate a dummy JETSCAPE file with random particle momenta.
```
Comment (Hendrik1704) [inline src/particle-analysis/GenerateFlow.py:223]: JETSCAPE -> OSCAR

File: src/particle-analysis/GenerateFlow.py [hunk 2]
```diff
@@ -217,9 +217,53 @@ def generate_dummy_JETSCAPE_file(self,output_path,number_events,multiplicity,see
...
+                            self.px_[particle] ** 2.
+                            + self.py_[particle] ** 2.
+                            + self.pz_[particle] ** 2.
+                            + mass ** 2.
+                        )
+                        output.write("%g %g %g %g %g %g %g %g  %g %d %d %d\n"
+                                    %(1,1,1,1,mass, energy,self.px_[particle],
+                                    self.py_[particle],self.pz_[particle],pdg, particle, 1))
+
+                output.write(f"# event {event} end 0 impact  -1.000 scattering_projectile_target no")
```
Comment (Hendrik1704) [inline src/particle-analysis/GenerateFlow.py:262]: There is a tab missing. This should be printed after each event.

**Review by Hendrik1704 — CHANGES_REQUESTED** [review]

**Comment by NGoetz** [comment]
I fixed the missing tab and change of name.

**Review by Hendrik1704 — APPROVED** [review]
Looks good

**Pull request merged**